Extension/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * Fix indexing of the entire root drive on Windows when no is folder open. [#2216](https://github.com/Microsoft/vscode-cpptools/issues/2216)
 * Disable the config provider message for headers outside the workspace and when debugging. [#2221](https://github.com/Microsoft/vscode-cpptools/issues/2221)
 * Add `Change Configuration Provider...` command. [#2224](https://github.com/Microsoft/vscode-cpptools/issues/2224)
+* Add a setting to silence configuration provider warnings. [#2292](https://github.com/Microsoft/vscode-cpptools/issues/2292)
```

Comment:
Can this be moved to the end, i.e. in issue# order? It makes it easier to find.

Reply:
I was grouping similar issues.  Are these sorted by id \#?

Reply:
It doesn't matter all that much to me.  I can move it.